openmc/filter.py

```diff
@@ -1332,6 +1332,35 @@ def check_bins(self, bins):
             cv.check_greater_than('filter value', v0, 0., equality=True)
             cv.check_greater_than('filter value', v1, 0., equality=True)
 
+    def get_tabular(self, values, interpolation='histogram'):
+        """Creates a openmc.stats.Tabular distribution using the EnergyFilter
+        bins and the provided values. Intended use is to help convert a
+        spectrum tally into a source energy.
+
+        Parameters
+        ----------
+        values : np.array
+            Array of numeric values, typically a tally.mean from a spectrum tally
+        interpolation : {'histogram', 'linear-linear', 'linear-log', 'log-linear', 'log-log'}
+            Indicate whether the density function is constant between tabulated
+            points or linearly-interpolated. Defaults to 'histogram'.         
+
+        Returns
+        -------
+        openmc.stats.Tabular
+            Tabular distribution with histogram interpolation
+        """
+
+        probabilities = values / sum(values)
+
+        probability_per_ev = probabilities / np.diff(self.bins).flatten()
```

Comment:
```suggestion
        probability_per_ev = probabilities / np.diff(self.values)
```

Reply:
ah yes, values instead of bins will probably fix the last comment you have as well. Thanks.